Pipfile

```diff
@@ -0,0 +1,10 @@
+[packages]
+
+[dev-packages]
+pytest = "*"
+pytest-cov = "*"
+pytest-flakes = "*"
+flake8 = "*"
+black = "*"
+mypy = "*"
```

Comment:
It would be useful to specify the major versions here - in case an update breaks the software later this would help understanding the source of the issue.